src/components/Toast/utils/ToasterConfig/ToasterConfig.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * ToasterConfig
+ * ------------------------------------------------
+ * DESCRIPTION_HERE
+ */
+
+import { Position } from "../../types"
+
+export interface Config {
+  duration: number
+  animationDuration: number
+  position: Position
+  behavior: "stack" | "replace"
+}
+
+export class ToasterConfig {
```

Comment:
I'd love to hear your reason for going with a class vs a factory function. Just curious—this looks like it does what's intended. 👌

Reply:
It was mostly an excuse to use a class, but the idea here was that it omits the possibility of passing an inline configuration object to the provider and ensures that a config object is always passed, which makes the developer more likely to know that it's an option to override those defaults...outside of documentation.

Reply:
If there is a better way without the use of a class, I'm totally open to it. I also realize it's not that big of a deal to pass an object inline, and for that matter you could set the object to a variable and pass in that variable, I just liked the patter that it forced.

Reply:
Yep, that makes sense. I think at least having something produce a default config is a good idea—but being able to pass a simple object sounds appealing as well. So maybe just a function that produces this config would be enough?

Reply:
I think if we go that route, I'd almost prefer just eliminating a util altogether and having the prop just define the object shape. It kinda goes against the pattern I was trying to create, but if we are going for simplicity/less code we can totally just delete all that.